README.md

```diff
@@ -128,17 +128,18 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
 
 The tool is provided as a unique `swiftgen` binary command-line, with the following subcommands:
 
-* `swiftgen assets [OPTIONS] DIR`
-* `swiftgen colors [OPTIONS] FILE`
-* `swiftgen fonts [OPTIONS] DIR`
-* `swiftgen storyboards [OPTIONS] DIR`
-* `swiftgen strings [OPTIONS] FILE`
+* `swiftgen colors [OPTIONS] FILE1 …`
+* `swiftgen fonts [OPTIONS] DIR1 …`
+* `swiftgen storyboards [OPTIONS] DIR1 …`
+* `swiftgen strings [OPTIONS] FILE1 …`
+* `swiftgen xcassets [OPTIONS] FILE1 …`
 
 Each subcommand has its own option and syntax, but some options are common to all:
 
 * `--output FILE` or `-o FILE`: set the file where to write the generated code. If omitted, the generated code will be printed on `stdout`.
 * `--template NAME` or `-t NAME`: define the Stencil template to use (by name, see [here for more info](https://github.com/SwiftGen/templates) to generate the output.
 * `--templatePath PATH` or `-p PATH`: define the Stencil template to use, using a full path.
+* Each command supports multiple input files (or directories where applicable).
 
 You can use `--help` on `swiftgen` or one of its subcommand to see the detailed usage.
```

Comment:
+ maybe add something like "You have to provide one and only one template when invoking swiftgen commands. (You should use either `-t` OR `-p`, but not BOTH at the same time"?